tools/install.py

```diff
@@ -15,32 +16,33 @@
 def register(command):
     command.help = "Install the system onto the target system"
 
-    # Configuration
-    command.add_argument("target", help="the target host to install the packages to")
+    command.add_argument("target", help="The target host or directory to install the packages to")
 
-    command.add_argument("--user", "-u", help="the user to install onto the target with")
+    command.add_argument("--local", "-l", dest="local", action="store_true", help="Install to a local directory")
```

Comment:
```suggestion
    command.add_argument("--local", "-l", dest="local", action="store_true", default=False, help="Install to a local directory")
```

Reply:
`False` [is the default](https://docs.python.org/3/library/argparse.html#action).